test/api/unit_roles_test.rb

```diff
@@ -46,4 +62,68 @@ def test_post_unit_roles_not_unique
     assert_equal to_post[:unit_id], last_response_body['unit_id']
     assert_equal to_post[:user_id], last_response_body['user_id']
   end
+
+  # PUT tests
+  # # Replace a unit role
+  # def test_put_unit_role
```

Comment:
Why commit this? If you dont want it delete it, otherwise uncomment it and get it working.